### PR TITLE
Structure changes and code refactoring

### PR DESCRIPTION
**Pull request by sanya-kenneth**
@RutaleIvanPaul
@CeciliaCaroline
I have reorganized my code and restructured my pages and files 
I have implemented email validation on the login.html page

### CODE REVIEW

**Comment by RutaleIvanPaul** [comment]
Zero conflicts,I guess you are ready for the merge.

File: UI/templates/authentication/login.html [hunk 1]
```diff
@@ -3,24 +3,24 @@
...
-    <div class="seg1"></div>
+    <div class="column1"></div>
 
-    <div class="seg2">
-        <img src="assets/img/logo.png">
+    <div class="column2">
+        <img src="../../assets/img/logo.png">
         <h1>Maintenance Tracker</h1>
 
         <div class="div_1">
```
Comment (CeciliaCaroline) [inline UI/templates/authentication/login.html]: rename class "div_1" to something more descriptive

File: UI/templates/user/create_request.html [hunk 2]
```diff
@@ -3,32 +3,32 @@
...
+        <li> <a href="../authentication/login.html">Signout</a></li>
     </ul>
 
     <br>
     <div class="div_1">
-        <img src="assets/img/logo.png">
+        <img src="../../assets/img/logo.png">
 
     </div>
     <div class="div_1">
```
Comment (CeciliaCaroline) [inline UI/templates/user/create_request.html]: Do as suggested above for class "div_1"

File: UI/templates/authentication/login.html (hunk 1 above)
Comment (sanya-kenneth) [reply UI/templates/authentication/login.html]: ok

File: UI/templates/user/create_request.html (hunk 2 above)
Comment (sanya-kenneth) [reply UI/templates/user/create_request.html]: ok let me do that right away

**Pull request merged**